src/sentry/integrations/jira/notify_action.py

```diff
@@ -160,15 +161,16 @@ def after(self, event, state):
         def create_issue(event, futures):
             """Create the Jira ticket for a given event"""
 
-            # TODO check if a Jira ticket already exists for the given event's issue. if it does, skip creating it
-            resp = installation.create_issue(self.data)
-            ExternalIssue.objects.create(
-                organization_id=organization.id,
-                integration_id=integration.id,
-                key=resp["key"],
-                title=event.title,
-                description=installation.get_group_description(event.group, event),
-            )
+            # HACK to get fixVersion in the correct format
+            if not isinstance(self.data["fixVersions"], list):
+                self.data["fixVersions"] = [self.data["fixVersions"]]
+
+            if self.data.get("dynamic_form_fields"):
+                del self.data["dynamic_form_fields"]
```

Comment:
had to delete this since https://github.com/getsentry/sentry/pull/21833 got merged, previously I wasn't sure what the data would look like before we sent it but now I know :)